packages/iov-ethereum/package.json

```diff
@@ -43,5 +43,8 @@
     "readonly-date": "^1.0.0",
     "rlp": "^2.2.0",
     "xstream": "^11.7.0"
+  },
+  "devDependencies": {
+    "@iov/ethereum": "^0.10.2"
```

Comment:
Now @iov/ethereum depends on @iov/ethereum ;) This needs to go into packages/iov-core

Reply:
Ha!, wrong package.json file